src/layouts/Main/components/AppNavigation/AppNavigation.js

```diff
@@ -145,6 +151,11 @@ const AppNavigation = ({
     setIsToggle(false);
   }, [isMD]);
 
+  useEffect(() => {
```

Comment:
@darwin808 I think we don't need the additional isLoggedIn condition because the <AppNavigation/> is already wrapped in isLoggedIn in Main.js and also other conditions in line `219` and `252` that depends on isLoggedIn. Please check if it will cause unusual behavior. Thanks